man/ctags-lang-python.7.rst.in

```diff
@@ -0,0 +1,167 @@
+.. _ctags-lang-python(7):
+
+==============================================================
+ctags-lang-python
+==============================================================
+-------------------------------------------------------------------
+Random notes about tagging python source code with Universal-ctags
+-------------------------------------------------------------------
+:Version: 0.0.0
+:Manual group: Universal-ctags
+:Manual section: 1
+
+SYNOPSIS
+--------
+|	**@CTAGS_NAME_EXECUTABLE@** ... --languages=+Python ...
+|	**@CTAGS_NAME_EXECUTABLE@** ... --language-force=Python ...
+|	**@CTAGS_NAME_EXECUTABLE@** ... --map-Python=+.py ...
+
+DESCRIPTION
+-----------
+This man page gathers random notes about tagging python source code.
+
+EXAMPLES
+--------
+This section shows how ctags uses kinds, roles, fields, and extras when tagging
+python source code for given input.
+
+Tagging ``import`` statements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+"input.py"
+.. code-block:: Python
+
+   import X0
+
+"output.tags"
+with "--options=NONE -o - --extras=+r --fields=+rzK input.py"
+.. code-block:: tags
+
+	X0	input.py	/^import X0$/;"	kind:module	roles:imported
+
+A tag for an imported module has ``module`` kind with ``imported`` role.  The
+module is not defined here; it is defined in another file. So the tag for the
+imported module is a reference tag; specify ``--extras=+r`` (or
+``--extras=+{reference}``) option for tagging it.  "roles:" field enabled with
+``--fields=+r`` is for recording the module is "imported" to the tag file.
+
+"input.py"
+.. code-block:: Python
+
+	import X1 as Y1
+
+"output.tags"
+with "--options=NONE -o - --extras=+r --fields=+rzK --fields-Python='+{nameref}' input.py"
+::
+
+	X1	input.py	/^import X1 as Y1$/;"	kind:module	roles:indirectlyImported
+	Y1	input.py	/^import X1 as Y1$/;"	kind:namespace	roles:def	nameref:module:X1
+
+"Y1" introduces a new name and is defined here. So "Y1" is tagged as a
+definition tag.  "X1" is imported in the way that its name cannot be used
+here. For letting know client tools that the name cannot be used,
```

Comment:
```suggestion
in this source file. For letting client tools know that the name cannot be used,
```

I think with "here", people may first think you are talking about `X1` can't be used as a tag name in the tag file.